src/services/ols-service.ts

```diff
@@ -0,0 +1,31 @@
+import { cancellableFetch } from '../cancellable-fetch';
+
+const AUTHORIZATIOMN_ENDPOINT = '/api/proxy/plugin/lightspeed-console-plugin/authorized';
+
+type AuthorizedObject = {
+  authorized: boolean;
+};
+type AuthorizationResponse = {
+  response: AuthorizedObject;
+};
+
+export const checkAuthorization = async (callback) => {
```

Comment:
This is probably a good case for using a React hook (`useIsAuthorized`), that could be called from `GeneralPage` like `const [isAuthorized] = useIsAuthorized()`

Reply:
fixed